.github/ISSUE_TEMPLATE/4-general-issue.yml

```
@@ -0,0 +1,39 @@
+name: General issue
+description: Create an issue for purposes not covered by the other templates
+labels: ["status: novel issue"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time in submitting an issue.
+
+        - This repository is maintained by volunteers, it may take a while for them to come back to you.
+        - Remember to include sufficient details and context.
+        - If you have multiple issue or suggestions, please submit them in separate issues.
+        - We accept pull request. If you know how to fix it - go ahead!
+  - type: textarea
+    id: describe
+    attributes:
+      label: Describe the issue
+      description: A clear and concise description of what the issue is.
+    validations:
+      required: true
+  - type: textarea
+    id: behavior
+    attributes:
+      label: Expected Behavior
+      description: If applicable, a clear and concise description of what you expect to happen in various scenario's.
+    validations:
+      required: false
+  - type: textarea
+    id: images
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain the issue.
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        _This issue will be read by people that are unable to look into your thoughts - this request is all they have to determine the viability of the feature. The more precise your request, the easier it becomes for us to help you._
```

```suggestion
        _This issue will be read by people that are unable to look into your thoughts - this request is all they have to determine the viability of the problem/suggestion. The more precise your request, the easier it becomes for us to help you._
```
I'm trying to write it to not be about a feature but the wording becomes kind of awkward.